kedro-airflow/README.md

```diff
@@ -1,5 +1,5 @@
 # Kedro-Airflow
-[![CircleCI](https://circleci.com/gh/quantumblacklabs/kedro-airflow/tree/master.svg?style=shield)](https://circleci.com/gh/quantumblacklabs/kedro-airflow/tree/master)
+[![CircleCI](https://circleci.com/gh/kedro-org/kedro-plugins/tree/main.svg?style=shield)](https://circleci.com/gh/kedro-org/kedro-plugins/tree/main)
```

Comment:
I'm wondering if it makes sense to keep this badge here if it's not specific to the plugin anymore. (same for the other readme.md files)

Reply:
Yeh, I agree feels a bit odd having it here now unless there is a way to make it specific to each plugin again?

Reply:
I don't think there's any way to make it specific to a plugin unfortunately. We can point the badge to a particular branch but not a particular CircleCI workflow. So I think just remove it.